css/header.css

```diff
@@ -0,0 +1,65 @@
+.header{
+    display:flex;
+    justify-content:space-between;
+    align-items:center;
+    height:55px;
+    background-color:#fff;
+    margin:10px;
+    padding:0 30px;
+    border:none;
+    border-radius:10px;
+    flex-wrap:wrap;
+  }
+  .logo{
+    font-size:27px;
+  }
+  .searchbar{
+    width:200px;
+    height:30px;
+    border:1px solid gray;
+    outline:none;
+    padding-left:10px;
+    border-radius:8px;
+  }
+  .header_nav ul{
+     display:flex;
+     justify-content:center;
+     align-items:center;
+  }
+  .header_nav ul li{
+     margin:10px;
+     list-style:none;
+     font-size:2.5rem;
+  }
+  .login_nav{
+      width:90px;
+      height:33px;
+      background-color:#8991a5;
+      color:#fff;
+      border:none;
+      border-radius:8px;
+      font-size:1.6rem;
+      margin:1rem;
+      text-align:center;
+      font-weight:700;
```

Comment:
use var

Reply:
yeah i will change to var  , thanks ranjeet